tests/manual/basic-styles.md

```diff
@@ -1,8 +1,9 @@
 ## Basic styles
 
 1. The data should be loaded with:
-  * italic "This",
-  * bold "editor",
+  * italic _"This"_,
+  * bold **"editor"**,
   * underline "instance",
-  * code "is an".
```

Comment:
You should also update `tests/manual/basic-styles.html` to reflect these changes.

Reply:
Done